packages/ui/__tests__/components/molecules/DoubleIcon.test.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {View} from 'react-native';
+import {shallow} from 'enzyme';
+import {DoubleIcon, Icon, lightTheme} from '@axelor/aos-mobile-ui';
+import {getGlobalStyles} from '../../tools';
+
+describe('DoubleIcon Component', () => {
+  const Colors = lightTheme.colors;
+
+  const props = {
+    topIconConfig: {
+      name: 'plus',
+    },
+    bottomIconConfig: {
+      name: 'user',
+    },
+  };
+
+  it('should render without crashing', () => {
+    const wrapper = shallow(<DoubleIcon {...props} />);
+
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('should give all props of topIconConfig and bottomIconConfig to the right icons', () => {
+    const topIconConfig = {
+      name: 'plus',
+      color: Colors.primaryColor.background,
+      size: 20,
+      touchable: false,
+      onPress: jest.fn(),
+      FontAwesome5: true,
+    };
+    const bottomIconConfig = {
+      name: 'user',
+      color: Colors.secondaryColor.background,
+      size: 15,
+      touchable: true,
+      onPress: jest.fn(),
+      FontAwesome5: false,
+    };
+    const wrapper = shallow(
+      <DoubleIcon
+        topIconConfig={topIconConfig}
+        bottomIconConfig={bottomIconConfig}
+      />,
+    );
+
+    expect(wrapper.find(Icon).at(0).props()).toMatchObject(bottomIconConfig);
+    expect(wrapper.find(Icon).at(1).props()).toMatchObject(topIconConfig);
+  });
+
+  it('should use size props if size is not defined in bottomIconConfig props', () => {
```

Comment:
[For V unless G wants to correct it too 👀] Replace title to "should use size props if size is not defined in icon configs" as for you test to be correct, both bottom and top should not have a size props.